app/Http/Controllers/Auth/AdmissionController.php

```diff
@@ -124,7 +124,7 @@ public function index(Request $request)
 
         $applications = $applications->with('user.educationalInformation')->distinct()->get()->sortBy('user.name');
 
-        if($request->input('return_excel')) {
+        if ($request->input('return_excel')) {
```

Comment:
argh, unrelated formatting changes

We should run the formatter at once on the whole code base, add it to `.git-blame-ignore-revs`, and then enjoy deepsource-autofix-free PRs